src/main/groovy/com/okta/test/mock/wiremock/TestUtils.groovy

```diff
@@ -39,4 +39,4 @@ class TestUtils {
         System.arraycopy(bigBytes, startSrc, resizedBytes, startDst, len);
         return resizedBytes;
     }
-}
+}
```

Comment:
Hah. Looks like we removed a newline here ;)

Reply:
Oops